contrib/macOS/resources/English.lproj/License.txt

```diff
@@ -0,0 +1,828 @@
+
+OpenZFS on OS X consists of software that is covered under the
+Common Development and Distribution License 1.0 and the Apple
+Public Source License 2.0.
```

Comment:
Are those actually compatible?
Why use a totally difference license for the OSX code? It doesn't really make sense to me.
We already have CDDL, GPL and BSD code mixed in, and now we also have 2(!?) different apple OSS licences?!

Reply:
Some files are used verbatim, like kextsymboltool.c - so its more of a "3rd party license" list I think. All code I have written personally is CDDL.

Reply:
Good to know, but how is this made clear in-code?